### PR TITLE
Split CWS Manage Store listing

### PR DESCRIPTION
**Pull request by AmySteam**
Part of #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by AmySteam** [comment]
@jpmedley I didn't change the content drastically. Mostly minor changes to match the pegasus redesign (screenshots and wording). I did not change the metrics page. I will do that later.

It would be nice to have this merged quickly this week so I can move to the next phase. Thanks so much!

File: site/en/docs/webstore/check-review/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+date: 2021-08-17
+updated: 2023-03-27
+description: >
+  How to check the review status of your Chrome Web Store item.
+---
+
+After [publishing][publish] your extension for the first time or [uploading][update] a new version, your item will go
+through a [review process][cws-review]. This page describes the how to check the status of your item .
+
+## The lifecycle of Chrome Web Store item
```
Comment (jpmedley) [inline site/en/docs/webstore/check-review/index.md:13]:
```suggestion
## The lifecycle of a Chrome Web Store item
```

File: site/en/docs/webstore/check-review/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+All Chrome Web Store items go through an automated [review process][cws-review]. In some instances, a manual review
+is required, especially when sensitive permissions are requested. For this reason, review times
+and/or approval times can take longer. Since a Chrome Web Store item goes through several stages,
+it's important to keep track of your item's status. See the lifecycle diagram below:
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/yakIoAzWEchO8urosQLs.png", alt="Diagram of the lifecycle of a Chrome Web Store item", width="800", height="177" %}
+
+## Checking your item's status
```
Comment (jpmedley) [inline site/en/docs/webstore/check-review/index.md:22]:
```suggestion
## Check your item's status
```

File: site/en/docs/webstore/check-review/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+If your extension has been determined to violate one or more terms or policies, you will receive an
+email notification that contains the violation description and instructions on how to rectify it. 
+
+If you did not receive an email within a week, check the status of your item (described above). If your item has
+been rejected, you can see the details on the **Status tab** of your item.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/C4wpnEeMriI9YeDAMDIr.png", alt="The Chrome Web Store
+Status Tab", width="700", height="276" %}
+
+If you have been informed about a violation and you do not rectify your item will be taken down. See
```
Comment (jpmedley) [inline site/en/docs/webstore/check-review/index.md:55]:
```suggestion
If you have been informed about a violation and you do not rectify it, your item will be taken down. See
```

File: site/en/docs/webstore/check-review/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+[Violation enforcement][enforcement] for more details.
+
+{% Aside %}
+
+To request further clarification on the reasons for the takedown or appeal the decision,
```
Comment (jpmedley) [inline site/en/docs/webstore/check-review/index.md:60]:
```suggestion
To request further clarification on the reasons for the takedown or to appeal the decision,
```

**Review by jpmedley — APPROVED** [review]
Please commit the nits before merging.

**Pull request merged**